include/device.h

```diff
@@ -208,6 +208,7 @@ extern "C" {
 
 struct device;
 
+typedef void (*device_pm_cb)(struct device *dev, int status);
```

Comment:
Could we add user-defined context the the callback?

Reply:
Ok